### PR TITLE
LPS-199251 Avoid showing horizontal scrollbar while opening sidebars

### PR DESCRIPTION
**Pull request by naraandrade**
Related ticket: https://liferay.atlassian.net/browse/LPS-199251

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey @naraandrade, good job with this PR. I just have one consideration.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+.lfr__objects-custom-vertical-bar--right-closed.c-slideout-end {
+	top: 129px;
+}
+
+.lfr-objects__model-builder-content {
+	display: flex;
+	height: calc(100vh - 129px);
+	overflow-x: hidden;
+	width: 100%;
+}
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.scss:11]: When we press the eye button to close the sidebars, I can see the sidebars go down a little before closing:

screenshot-1697569450


To resolve this small behavior, we can replace the uses of 129px with 121 px.

```
.lfr__objects-custom-vertical-bar--left-closed.c-slideout-start,
.lfr__objects-custom-vertical-bar--right-closed.c-slideout-end {
	top: 129px;
}

.lfr-objects__model-builder-content {
	display: flex;
	height: calc(100vh - 129px);
	overflow-x: hidden;
	width: 100%;
}
```

Comment (naraandrade) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.scss:11]: @igor-franca Thanks for noticing that! I'll apply the fix :)

**Comment by naraandrade** [comment]
ci:test:sf

**Review by igor-franca — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by victorhcunha** [comment]
ci:test:bundle

**Comment by victorhcunha** [comment]
There are many failures reported on CI, but they don't seem to be related to the modifications (a change in the creation of objects via API in our tests is defining the value of the object's folder even with the feature flag deactivated).

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward:force

**Comment by victorhcunha** [comment]
The stable failure is not related to PR changes.

**Comment by victorhcunha** [comment]
ci:reevaluate:1287401_7959

**Comment by naraandrade** [comment]
ci:forward:force

**Pull request closed without merging**